### ISSUE TITLE
Export Client Configuration

### ISSUE THREAD

**Issue by rosenbach** [skoruba/Duende.IdentityServer.Admin#197]
**Is your feature request related to a problem? Please describe.**

There might be some situation that you want to backup your client configuration. Or you want to import it in the future again, or somewhere else.

**Describe the solution you'd like**

I would like to have an opportunity to export a client configuration.

**Describe alternatives you've considered**

Maybe a more complex export button would be nice that would either enable a) copying to the clipboard or b) downloading the current client configuration.

**Additional context**

I am preparing a PR for this right now, furthermore I am preparing another feature request for importing client configurations.

**Comment by skoruba** [comment]
Hey @rosenbach - that is great idea, I fully agree that we need something like this.
What about also import feature? I thing we need also add some version into export json file, for import feature as well.

**Comment by skoruba** [comment]
Sorry, I missed your issue #199. Would be great if you've already worked on the import feature and you can send a PR. 👍

**Comment by rosenbach** [comment]
Hi @skoruba 
thanks for your anwers
Can you specify what you mean with we need some version in the export json file? You mean a versioning of the exported client configuration?

Regarding the import feature:
as you already mentioned, I created a seperate issue for that. I already see quite some value by having the export button. The import feature is not so trivial when thinking about how to make a viable minimal product. We can gladly discuss this in the other issue #199

**Comment by skoruba** [comment]
> Can you specify what you mean with we need some version in the export json file? You mean a versioning of the exported client configuration?

Exactly, I thought the export would contain a version of the client configuration and we can add some conditions in the future in case the data structure changes.
